static/locales/_en.json

```diff
@@ -280,15 +280,75 @@
     "popup_status": "Status",
     "popup_online": "Online",
     "popup_offline": "Offline",
-    "popup_invalid_iv_filter": "Invalid IV Filter",
-    "popup_invalid_color_value": "Invalid Color Value",
-    "popup_invalid_nest_filter": "Invalid Nest",
-    "popup_invalid_candy_filter": "Invalid Candy Filter",
-    "popup_invalid_stardust_filter": "Invalid Stardust Filter",
+    "popup_invalid_iv_filter": "Invalid IV Filter!",
+    "popup_invalid_color_value": "Invalid Color Value!",
+    "popup_invalid_nest_filter": "Invalid Nest Filter!",
+    "popup_invalid_candy_filter": "Invalid Candy Filter!",
+    "popup_invalid_stardust_filter": "Invalid Stardust Filter!",
     "tiny": "Tiny",
     "small": "Small",
     "normal": "Normal",
     "large": "Large",
     "big": "Big",
-    "default_stuff": "Default"
-  }
\ No newline at end of file
+    "weather_none": "None",
+    "weather_clear": "Clear",
+    "weather_rain": "Rain",
+    "weather_partly_cloudy": "Partly Cloudy",
+    "weather_cloudy": "Cloudy",
+    "weather_windy": "Windy",
+    "weather_snow": "Snow",
+    "weather_fog": "Fog",
+    "fire": "Fire",
+    "grass": "Grass",
+    "ground": "Ground",
+    "water": "Water",
+    "electric": "Electric",
+    "bug": "Bug",
+    "rock": "Rock",
+    "fairy": "Fairy",
+    "fighting": "Fighting",
+    "poison": "Poison",
+    "dragon": "Dragon",
+    "flying": "Flying",
+    "psychic": "Psychic",
+    "ice": "Ice",
+    "steel": "Steel",
+    "dark": "Dark",
+    "ghost": "Ghost",
+    "common": "Common",
+    "uncommon": "Uncommon",
+    "rare": "Rare",
+    "ultra_rare": "Ultra Rare",
+    "regional": "Regional",
+    "event": "Event",
+    "kanto": "Kanto",
+    "johto": "Johto",
+    "hoenn": "Hoenn",
+    "sinnoh": "Sinnoh",
+    "unova": "Unova",
+    "kalos": "Kalos",
+    "alolan": "Alolan",
+    "galarian": "Galarian",
+    "ways_to_filter": "Ways to Filter",
+    "general_basics": "General Basics",
+    "iv_range": "IV Range",
+    "iv_example": "90-100% IV Pokemon",
+    "level_range": "Level Range",
+    "level_example": "Level 30 to 35 Pokemon",
+    "stats": "Stats",
+    "stats_example": "Pokemon with 0-1 Attack, 14 Defense, & 15 Stamina",
+    "cp_example": "All Pokemon with CP of 2000-3000",
+    "not": "NOT",
+    "not_example": "All Pokemon that do not have their IVs scanned",
+    "pvp": "PVP",
+    "pvp_example": "Pokemon that are rank 1-3 for either Great League or Ultra League",
+    "stringing_conditions_together": "Stringing Conditions Together",
+    "or": "or",
```

Comment:
`NOT` and help strings logic gates are already upper case. Lets make them all upper.

Small bonus with `On` and `Off`. 

It's much easier to read them like that I think.
```
    "filter_and": "And",
    "filter_or": "Or",
    "filter_on": "On",
    "filter_off": "Off",
    "or": "or",
```